compiler/src/typed/translprim.re

```diff
@@ -1475,43 +1473,12 @@ let transl_prim = (env, desc) => {
     pla_loc: Location.dummy_loc,
   };
 
-  // `attrs` are attributes which should be applied to the `let` which gets implicitly generated.
```

Comment:
This comment is no longer useful?

Reply:
Nope. It's mostly about the static pointers that this PR removes. The other bits about the attributes are just wrong but this comment wasn't updated when the logic changed.